zookeeper-client/zookeeper-client-c/pom.xml

```diff
@@ -28,26 +28,21 @@
   </parent>
 
   <artifactId>zookeeper-client-c</artifactId>
-  <packaging>jar</packaging>
```

Comment:
was this actually putting the c client in a jar file before?

Reply:
No. Most of the plugins that are bound to the lifecycle by default for the jar packaging are smart enough to observe that there was no work for them to do. However, it is still wasteful to have these plugins execute at all. Changing the packaging type stops those plugins from executing pointlessly, because they are not bound to the lifecycle in the first place.